src/routes/api/users/auth.js

```diff
@@ -53,6 +53,53 @@ class User {
       });
     }
   }
+
+  // static async socialAuth(userObject, isRegistered) {
+  //   try {
+  //     if (isRegistered) {
+
+  //     }
+  //   } catch (err) {
+
+  //   }
+  // }
+
+  /**
+ * method for checking if user is already registered
+ * @param {String} email user email
+ * @param {String} username user email
+ * @returns {Boolean} query result
+ * @memberof User
+ */
+  static async socialAuth(email, username) {
```

Comment:
socialAuth middleware method replicates the actions of the userSignup actions. find a way to interact with the userSignup middleware

Reply:
Yes. This is because the UserSignup method returns an object but with social login, the goal is to redirect to another URL with the user token as a parameter in the URL. A chore would be created after this to factor out the functionality of adding a user to the database from the UserSignup method. With this, the socialAuth method and the UserSignup method would call a single function to add a user to the database